images/dcos/Dockerfile

```diff
@@ -1,25 +1,32 @@
-FROM busybox:ubuntu-14.04
+FROM alpine:3.2
```

Comment:
we've experienced lots of inconsistency w/ respect to dns resolution across different base containers. curious how this choice will impact us.

/cc @spacejam @karlkfi